Bottom: c5a9ce2d08e3d456c3d0ca15114901becddb0f4b
Top:    6e36994583b429b0c761d46e17c56430e523a1d6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:39:18 +0000

Remove unused parameter SetDefaultNames from CreateNewPCB

The SetDefaultNames parameter to CreateNewPCB was not used (despite various
callers passing both true and false), so remove it.


---

diff --git a/src/action.c b/src/action.c
index 3a36c79..86b379d 100644
--- a/src/action.c
+++ b/src/action.c
@@ -5938,7 +5938,7 @@ ActionNew (int argc, char **argv, Coord x, Coord y)
 	SaveInTMP ();
       RemovePCB (PCB);
       PCB = NULL;
-      PCB = CreateNewPCB (true);
+      PCB = CreateNewPCB ();
       CreateNewPCBPost (PCB, 1);
 
       /* setup the new name and reset some values to default */
diff --git a/src/buffer.c b/src/buffer.c
index 7067b8d..9d102f4 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -1121,7 +1121,7 @@ ConvertBufferToElement (BufferType *Buffer)
 bool
 LoadLayoutToBuffer (BufferType *Buffer, char *Filename)
 {
-  PCBType *newPCB = CreateNewPCB (false);
+  PCBType *newPCB = CreateNewPCB ();
 
   /* new data isn't added to the undo list */
   if (!ParsePCB (newPCB, Filename))
diff --git a/src/create.c b/src/create.c
index a7185df..6da9271 100644
--- a/src/create.c
+++ b/src/create.c
@@ -140,7 +140,7 @@ pcb_colors_from_settings (PCBType *ptr)
  * creates a new PCB
  */
 PCBType *
-CreateNewPCB (bool SetDefaultNames)
+CreateNewPCB (void)
 {
   PCBType *ptr;
   int i;
diff --git a/src/create.h b/src/create.h
index e9c2697..f49f6d0 100644
--- a/src/create.h
+++ b/src/create.h
@@ -39,7 +39,7 @@ void CreateBeLenient (bool);
 
 DataType * CreateNewBuffer (void);
 void pcb_colors_from_settings (PCBType *);
-PCBType * CreateNewPCB (bool);
+PCBType * CreateNewPCB ();
 /* Called after PCB->Data->LayerN is set.  Returns zero if no errors,
    else nonzero.  */
 int CreateNewPCBPost (PCBType *, int /* set defaults */);
diff --git a/src/file.c b/src/file.c
index 0772139..8c3dea3 100644
--- a/src/file.c
+++ b/src/file.c
@@ -386,7 +386,7 @@ real_load_pcb (char *Filename, bool revert)
 {
   const char *unit_suffix, *grid_size;
   char *new_filename;
-  PCBType *newPCB = CreateNewPCB (false);
+  PCBType *newPCB = CreateNewPCB ();
   PCBType *oldPCB;
 #ifdef DEBUG
   double elapsed;
diff --git a/src/main.c b/src/main.c
index a6b22ad..04a116c 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1981,7 +1981,7 @@ main (int argc, char *argv[])
     }
 
   /* Create a new PCB object in memory */
-  PCB = CreateNewPCB (true);
+  PCB = CreateNewPCB ();
   ParseGroupString (Settings.Groups, &PCB->LayerGroups, &PCB->Data->LayerN);
   /* Add silk layers to newly created PCB */
   CreateNewPCBPost (PCB, 1);
